Wiki/Programs/PokemonSV/EggFetcher.md

```diff
@@ -1,48 +1,57 @@
 # Egg Fetcher
```

Comment:
Can you add a note here that this program has changed significantly in the not-yet-released code?

Reply:
Update note added, along with adding a little more clarity to instructions.